package.json

```diff
@@ -40,5 +42,6 @@
     "lodash": "^4.17.21",
     "node-uuid": "^1.4.3",
     "slf": "^1.0.0"
-  }
+  },
+  "prettier": "prettier-config-surikaterna"
```

Comment:
We should not specify it both here and in the prettierrc. Remove this, keep the file.